app/views/applications/show.html.haml

```diff
@@ -14,7 +14,7 @@
   #application
     %article#application-info
       %header
-        %h3.address= @application.address
+        %h1.address= @application.address
```

Comment:
This change is related to a different issue that #826 so it shouldn't be part of this PR (that would be an awesome issue to fix in a separate PR though 🎉 ).